govuk_content_api.rb

```diff
@@ -89,43 +89,11 @@ def set_expiry(duration = DEFAULT_CACHE_TIME, options = {})
 
   get "/local_authorities.json" do
     set_expiry LONG_CACHE_TIME
```

Comment:
Just realised that you left this line here but removed it from the other method. Was that intentional?